code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -55,7 +55,13 @@
 		return FALSE
 
 	if(package)
-		to_chat(M, SPAN_WARNING("How do you expect to eat this with the package still on?"))
+		if(user.a_intent == INTENT_HARM)
+			return ..() // chunk box gaming
+
+		if(user == M)
+			to_chat(M, SPAN_WARNING("How do you expect to eat this with the package still on?"))
+		else
+			to_chat(M, SPAN_WARNING("[M] made an endearing attempt to force feed you a snack with the packaging still on."))
```

Comment:
The person being clicked on here is `M`, so the message should refer to `user` instead:
```suggestion
			to_chat(M, SPAN_WARNING("[user] made an endearing attempt to force feed you a snack with the packaging still on."))
```